libminifi/include/FlowFileRecord.h

```diff
@@ -39,6 +39,7 @@
 #include "ResourceClaim.h"
 #include "Connection.h"
 #include "io/OutputStream.h"
+#include "io/StreamPipe.h"
```

Comment:
Is this a leftover?

Reply:
intentional as I didn't want to add the includes, as this change of moving `*StreamCallback` to a separate file is already tangential to the issue, I would like to do it in a separate PR, likewise with moving it to the `io` namespace
[the ticket](https://issues.apache.org/jira/browse/MINIFICPP-1386)